lib/linguist/languages.yml

```diff
@@ -3082,6 +3082,21 @@ IGOR Pro:
   tm_scope: source.igor
   ace_mode: text
   language_id: 162
+ImHex Pattern Language:
+  type: programming
+  color: "#3a6be0"
+  extensions:
+  - ".hexpat"
+  aliases:
+  - ImHex
+  - PatternLanguage
+  - Pattern Language
+  - pl
```

Comment:
ImHex Pattern Language is _one_ pattern language. There are others which haven't been added but may be in future. As such these three are too generic to be associated with a single language.

```suggestion
```